<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My System Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Host</string>
     </attribute>
     <widget class="QListWidget" name="system">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>401</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="runtime">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>301</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Progress</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>821</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>510</y>
        <width>741</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%p Is Running</string>
      </property>
     </widget>
     <widget class="QLabel" name="RunNum">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>510</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="question">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>   Let's Search ~ v ~</string>
      </property>
     </widget>
     <widget class="QLabel" name="warn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(239, 41, 41)</string>
      </property>
      <property name="text">
       <string>! NoMatch</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>End It QAQ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Turn Off 88~</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Performance</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>551</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>400</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>120 seconds</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>460</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>430</y>
          <width>151</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Maximum speed:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>% Ultilization</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>400</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>430</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>2.3 Mhz</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>490</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Logical processors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="ult">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>470</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>0</y>
          <width>361</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Inter(R) Core(TM) i5-6300H Processor</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>440</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Ultilization</string>
        </property>
       </widget>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>571</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>40</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>460</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Physical processors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>490</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>-10</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>      CPU</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot_2" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>571</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="mem">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>470</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Memory</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Memory usage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>3.7GB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>50</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>3.7GB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>400</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>400</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>120 seconds</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>440</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>In Use</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Swap</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot_3" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>571</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sw">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>470</y>
          <width>151</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>20</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>3.8GB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>50</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>3.7GB</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SWAP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Swap usage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>400</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>120 seconds</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>440</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>In Use</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>400</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="curTime">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>580</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="curCpu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="curMem">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="curCpuIm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="curMemIm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
